doc/getting-started/rt_industry_ubuntu.rst

```diff
@@ -223,7 +223,7 @@ Build and install the ACRN kernel
 
    .. code-block:: none
 
-      $ git checkout -b v2.2 remotes/origin/release_2.2
+      $ git checkout v2.3 
```

Comment:
Will this tag be created later? Or should the user check the `release_2.3` branch?

Reply:
v2.3 will be the "official release" tag, so we should be using that.